### DISCUSSION TITLE
A config file for my modules

### DISCUSSION THREAD

**Discussion by shykes** [dagger/dagger#12463] [Ideas]
Now that modules have constructors, we have a way to reduce the verbosity of calling them from the command-line: write module configuration to a config file.

**Comment by shykes** [comment]
This is relevant to #6112

**Comment by shykes** [comment]
As discussed with @aluzzardi today :)

**Comment by shykes** [comment]
Note, at least one user has developed a wrapper tool to do this.

**Comment by jedevc** [comment]
Ooh this seems similar to https://github.com/dagger/dagger/issues/8419, but I remember when opening it I couldn't find this issue 😂

**Comment by marcosnils** [comment]
Shall we close #8419 and consolidate here @jedevc ?

**Comment by shykes** [comment]
A real-world example of a user implementing a stopgap: https://discord.com/channels/707636530424053791/1324306231875338300
